.travis.yml

```diff
@@ -17,6 +17,7 @@ install:
     - pip install wheel
     - pip install pysam>=0.11.2
     - pip install -r requirements.txt
+    - pip install .
```

Comment:
Even with adding the `env` make target as a prerequisite to `make testall`, the extension module wouldn't import properly until I added this line.